test/integration/cli-domain-package-static-files-minify-file.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var expect = require('chai').expect;
+
+describe('cli : domain : package-static-files : minify-file', function(){
+	var minifyFile = require('../../src/cli/domain/package-static-files/minify-file');
+
+	describe('when minifying .js file', function(){
+
+		describe('when file contains es6', function(){
+
+			var content = 'const hi = (name) => `hello ${name}`;';
+			var minified;
+
+			before(function(){
+				minified = minifyFile('.js', content);
+			});
+
+			it('should minify it', function(){
```

Comment:
I think that the process is referentially transparent , so it should return always the same result